### ISSUE TITLE
law.com: nuisance

### ISSUE THREAD

**Issue by john-clark** [uBlockOrigin/uAssets#14672]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.law.com/dailyreportonline/2022/08/19/ga-jury-reaches-16-8m-verdict-against-ford-with-punitive-damages-phase-to-come/?slreturn=20220804123146`

### Category

nuisance

### Describe the issue

overlay

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.44.0
Firefox: 104
filterset (summary): 
  network: 81807
  cosmetic: 41804
  scriptlet: 17803
  html: 741
listset (total-discarded, last updated): 
  default: 
    user-filters: 2-0, never
    ublock-filters: 33219-37, now
    ublock-badware: 4206-0, now
    ublock-privacy: 263-0, now
    ublock-abuse: 76-0, now
    ublock-unbreak: 1861-0, now
    ublock-quick-fixes: 427-0, now
    easylist: 64917-597, now
    easyprivacy: 27457-614, now
    urlhaus-1: 7465-0, now
    plowe-0: 3671-3, now
filterset (user): [array of 2 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 181 ms (selfie)
  maxAssetCacheWait: 124 ms
popupPanel: 
  blocked: 22
  blockedDetails: 
    law.com: 5
    chartbeat.com: 1
    demdex.net: 2
    doubleclick.net: 2
    dpmsrv.com: 1
    hotjar.com: 1
    licdn.com: 1
    ml314.com: 2
    moatads.com: 2
    sail-horizon.com: 2
    tealiumiq.com: 3
```
</details>

**Comment by mapx-** [comment]
add

```
law.com###reg-modal
law.com##.show.fade.modal-backdrop
law.com##body:style(overflow: auto !important;)
```

@ryanbr

**Comment by MasterKia** [comment]
@mapx- Force signup/login doesn't get fixed in uBlock Annoyances?

**Comment by mapx-** [comment]
No, and FA annoyances has a uBO section so better have the filters there

**Comment by ryanbr** [comment]
Seems to be a paywall?

**Comment by MasterKia** [comment]
Last I checked it said "register for free".
